src/app/public/modules/split-view/split-view-workspace.component.ts

```diff
@@ -77,14 +78,16 @@ export class SkySplitViewWorkspaceComponent implements AfterViewInit, OnDestroy
     private splitViewService: SkySplitViewService
   ) {}
 
-  public ngAfterViewInit(): void {
+  public ngOnInit(): void {
     this.splitViewService.isMobileStream
       .pipe(takeUntil(this.ngUnsubscribe))
       .subscribe((mobile: boolean) => {
         this.isMobile = mobile;
         this.changeDetectorRef.markForCheck();
       });
+  }
```

Comment:
☝️ Not a part of the scope of this PR, but `isMobile` was throwing an Expression Changed error if you loaded the page starting out in a small window. Moving the check to `ngOnInit` fixes the error.